src/app/components/InfoLink/InfoLink.tsx

```diff
@@ -3,18 +3,29 @@ import Typography from '../Typography/Typography';
 import type { ReactNode } from 'react';
 import { Link } from 'react-router-dom';
 import style from './InfoLink.module.css';
+import ForwardIcon from '../../assets/SVG/Forward';
 
 type InfoLinkProps = {
+  type?: 'small' | 'big';
   children: ReactNode;
   className?: string;
   to: string;
 };
 
-function InfoLink({ children, className, to }: InfoLinkProps): JSX.Element {
+function InfoLink({
+  type = 'small',
+  children,
+  className,
+  to,
+}: InfoLinkProps): JSX.Element {
   return (
     <Link to={to} className={className}>
-      <Typography className={style.info} size={'xsAction'}>
+      <Typography
+        className={`${style.info} ${type === 'big' && style.big}`}
```

Comment:
If you had a class with the name "small" this component could adjust styles like so:
`className={`${style.info} ${style[type]}`}`
Actually, you could probably do that anyway